save-frontend/src/main/kotlin/com/saveourtool/save/frontend/externals/progressbar/ReactCircleBuilder.kt

```diff
@@ -11,6 +11,7 @@ import react.react
 enum class Color(val hexColor: String) {
```

Comment:
Suggestion to make `Color` private if it is used only in this file as we have many similar classes (if I am not mistaken)

Reply:
There is no such class. I just think that it can be used in other places in the future

Reply:
Oh ok